### PR TITLE
Javascript/javascript1/week3

### PR DESCRIPTION
**Pull request by dinythomas89**

### CODE REVIEW

File: javascript/javascript1/week3/index.js [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+const travelInformation = {
+    speed: 50,
+    destinationDistance: 432,
+}; 
+function getTime(travelInfo) {
+    const time=travelInfo.destinationDistance/travelInfo.speed;
+    const hours=parseInt(Number(time));
+    const minutes=Math.round((Number(time)-hours) * 60);
+    return hours+' hours and '+minutes+' minutes';
+}
```
Comment (cameronjdavis) [inline javascript/javascript1/week3/index.js]: Consistent formatting is important, Google for "vs code format document" to learn how to format your document consistently.

File: javascript/javascript1/week3/index.js [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+      days: 2,
+      hours: 19,
+      minutes: 0,
+    },
+];
+const totalMinutesIn80Years= 80*525600;
+function logOutSeriesText() {
+    let totalDurationOfSeries=0;
+    for(let series of seriesDurations){
+        const seriesDurationInMinutes=(series.days*1440)+(series.hours*60)+series.minutes;
```
Comment (cameronjdavis) [inline javascript/javascript1/week3/index.js]: I don't know what 1440 is just from looking at it. I guess it is number of minutes in a day. Create a const to give the next coder a clue of what it means.
```
const minutesInDay = 1440;
```

File: javascript/javascript1/week3/smart-ease.js [hunk 3]
```diff
@@ -0,0 +1,30 @@
+//save a note
+const notes=[];
+function saveNote(content, id) {
+    const resultNote={content:content, id:id};
```
Comment (cameronjdavis) [inline javascript/javascript1/week3/smart-ease.js]: A shortcut for this is.
```suggestion
    const resultNote={content, id};
```

**Review by cameronjdavis — CHANGES_REQUESTED** [review]

File: javascript/javascript1/week3/index.js (hunk 1 above)
Comment (dinythomas89) [reply javascript/javascript1/week3/index.js]: Did formatting for all the js pages

File: javascript/javascript1/week3/index.js (hunk 2 above)
Comment (dinythomas89) [reply javascript/javascript1/week3/index.js]: created const minutesInDay = 1440;

File: javascript/javascript1/week3/smart-ease.js (hunk 3 above)
Comment (dinythomas89) [reply javascript/javascript1/week3/smart-ease.js]: Modified the code by giving the shortcut

**Pull request merged**